astore/client/auth/auth.go

```diff
@@ -74,14 +74,18 @@ func (c *Client) Login(username, domain string, o LoginOptions) (string, error)
 	}
 	fmt.Printf("\t%s\n\nTo complete authentication with @%s.\n"+
 		"I'll be waiting for you, but hurry! The request may timeout.\nHit Ctl+C with no regrets to abort.\n", ares.Url, domain)
-	browser.OpenURL(ares.Url)
-	pubKey, privateKey, err := kcerts.MakeKeys(kcerts.GenerateDefault)
+	go func() {
+		if err := browser.OpenURL(ares.Url); err != nil {
+			o.Logger.Infof("error opening url, you must open it yourself")
+		}
+	}()
+	privateKey, pubKey, err := kcerts.MakeKeys(kcerts.GenerateDefault)
```

Comment:
I changed MakeKeys in my PR to follow the same convention as other library functions, now returns `pubKey, privateKey, err`, very unlucky that none of the uses below caused the compiler to freak out.